extensions/wikia/CategoryPage3/CategoryPage3Hooks.class.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+class CategoryPage3Hooks {
+
+	const COOKIE_NAME = 'category-page-layout';
+	const GLOBAL_PREFERENCE_NAME = 'category-page-layout';
+
+	private static $oldQueryParams = [
+		'display',
+		'filefrom',
+		'fileuntil',
+		'page',
+		'pagefrom',
+		'pageuntil',
+		'sort',
+		'subcatfrom',
+		'subcatuntil',
+	];
+
+	/**
+	 * @param $categoryInserts
+	 * @param $categoryDeletes
+	 * @return bool
+	 * @throws MWException
+	 */
+	public static function onAfterCategoriesUpdate( $categoryInserts, $categoryDeletes/*, $title*/ ): bool {
```

Comment:
do we need this commented-out param?

Reply:
I think that's a convention (one of many we have), in case someone wanted to do something with this param in the future.